remill/BC/Lifter.cpp

```diff
@@ -284,7 +284,7 @@ llvm::Value *InstructionLifter::LiftShiftRegisterOperand(
         context, op.shift_reg.extract_size);
 
     if (reg_size > op.shift_reg.extract_size) {
-      curr_size = op.shift_reg.extract_size;
+      curr_size = llvm::TypeSize::Fixed(op.shift_reg.extract_size);
```

Comment:
This API isn't available prior to LLVM 10, you'll need to wrap it in some kind of compatibility code.

Reply:
done